templates/quickstart-crowd-dc.template.yaml

```diff
@@ -581,7 +569,7 @@ Parameters:
       It cannot start or end with a hyphen (-).
     Type: String
   QSS3KeyPrefix:
-    Default: 'quickstart-atlassian-jira/'
+    Default: 'quickstart-atlassian-crowd/'
```

Comment:
The `S3 bucket` prefix, `quickstart-atlassian-crowd`, will need to be created in the bucket `aws-quickstart`

Reply:
@nanux @bpar476 do you know who I could reach out to for assistance on this?

Reply:
See my comment in the previous template